src/ensembl/src/content/app/browser/Browser.tsx

```diff
@@ -3,7 +3,8 @@ import React, {
   useCallback,
   useRef,
   useEffect,
-  Fragment
+  Fragment,
```

Comment:
Not that it's important, but can we please remove the fragment from jsx below? It doesn't seem to be doing anything useful here.